README.md

```diff
@@ -16,7 +16,15 @@ about more important problems than building a file uploader.
 **[Read the docs](https://uppy.io/docs)** |
 **[Try Uppy](https://uppy.io/examples/dashboard/)**
 
-<a href="https://transloadit.com" target="_blank"><img width="185" src="https://github.com/transloadit/uppy/raw/main/assets/developed-by-transloadit.png"></a>
+<p>
+  <a href="https://transloadit.com" target="_blank">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://github.com/transloadit/uppy/assets/375537/6651e57e-cb57-4336-8745-6473ae68d0bd">
+      <source media="(prefers-color-scheme: light)" srcset="https://github.com/transloadit/uppy/assets/375537/7f14421d-1e37-464e-8203-ade121216c88">
+      <img src="https://github.com/transloadit/uppy/assets/375537/7f14421d-1e37-464e-8203-ade121216c88" alt="Developed by Transloadit">
+    </picture>
```

Comment:
Shouldn't we be using SVG instead, and have that switch in the CSS of the SVG?

Reply:
I also followed GitHub's own recommendation, which was posted later (https://github.blog/changelog/2022-05-19-specify-theme-context-for-images-in-markdown-beta/). This code works well with the GitHub setting — I've just tested it via keeping the system theme light but setting the dark theme in GitHub Appearance settings.

<img width="420" src="https://github.com/transloadit/uppy/assets/375537/5a719c8d-8952-4152-a0ac-8678dcda5566"><br>

Both images are already SVGs (GitHub just hid their extension after upload for some reason).

CSS in SVG didn't work on GitHub, at least when I had a similar problem around half a year ago.